boot/zephyr/Kconfig

```diff
@@ -412,6 +412,13 @@ config MCUBOOT_CLEANUP_RAM
 	help
 	  Sets contents of memory to 0 before jumping to application.
 
+config MCUBOOT_LOOP_AFTER_RAM_CLEANUP
+	bool "Loop after RAM cleanup"
+	depends on MCUBOOT_CLEANUP_RAM
+	help
+	  Debug option that keeps execution in infinite loop after RAM cleanup
```

Comment:
Shouldn't this be called "test" or "verification" option, and Kconfig named accordingly?